packages/affiliates/config.example.js

```diff
@@ -0,0 +1,24 @@
+// This is an example file for how to run deployer rewards.  Use this as template to adjust parameters for your own run
+// example: node apps/DeployerRewards ./config.example.js >> output.json
+const moment = require("moment");
+
+// emps to look at
+const empWhitelist = ["0xaBBee9fC7a882499162323EEB7BF6614193312e3", "0x3605Ec11BA7bD208501cbb24cd890bC58D2dbA56"];
+
+// start time and endtime in ms timestamp
+const startTime = moment("2020-9-20 23:00:00", "YYYY/MM/DD  HH:mm z").valueOf(); // utc timestamp
+const endTime = moment("2020-10-19 23:00:00", "YYYY/MM/DD HH:mm z").valueOf();
+
+// 10000 rewards to split across all emp creators
+const totalRewards = 10000n.toString();
+
+// network 1 = mainnet, 42 = kovan, will default to 1 if not specified
+const network = 1;
```

Comment:
The `empWhitelist` hard coded addresses would not work with kovan right?

Reply:
yea the idea is that the user should create their own config file with the right values and the network they want to test.